contracts/utils/ExponentialNoError.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
https://en.wikipedia.org/wiki/BSD_licenses

"The BSD license is a simple license that merely requires that all code retain the BSD license notice if redistributed in source code format, or reproduce the notice if redistributed in binary format. The BSD license (unlike some other licenses e.g. [GPL](https://en.wikipedia.org/wiki/GPL)) does not require that source code be distributed at all.

seems okay to me, thoughts @CyclopsRex ?

Reply:
License seems OK. I want to take a closer look at how this is being used, given we already have two libraries for working with decimals represented as wad so it would be nice to avoid a third.